test/test_helper.rb

```diff
@@ -2,109 +2,25 @@
 require 'rails/test_help'
 require 'nokogiri'
 require 'faker'
+require 'vcr'
```

Comment:
I guess we don't need nokogiri or faker anymore in this file, that was only used for `stub_random_search`, right?